internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/CoordinatorGrpcClient.java

```diff
@@ -351,7 +351,7 @@ public RssFetchClientConfResponse fetchClientConf(RssFetchClientConfRequest requ
       rpcResponse =
           blockingStub
               .withDeadlineAfter(request.getTimeoutMs(), TimeUnit.MILLISECONDS)
-              .fetchClientConf(Empty.getDefaultInstance());
+              .fetchClientConfV2(request.toProto());
```

Comment:
Why do we need `v2`?

Reply:
fetchClientConf uses the empty request, it's hard to extend for this PR.